css/default.css

```diff
@@ -940,8 +940,12 @@ span.partial_res_expln{
 cursor: pointer;
 }
 
-/*QR COde Fix*/
+/*Fix QR Code display*/
 
 .ui-dialog.ui-widget.ui-widget-content.ui-corner-all.ui-front.ui-draggable.ui-resizable {
   margin-top: 100px;
+  position: absolute; 
+  height: 180.016px; 
+  width: 160px; 
+  inset: 701.969px auto auto 1627px;
```

Comment:
Do these values really require thousandths-of-pixel precision? Probably doesn't hurt anything, but feels very odd.

Reply:
At that point, yes. Inset values are now percentages following DIV location change

Reply:
Height is still `180.016px`.  I cannot think of any reason that would be needed instead of `180px` (or even `181px`), nor why inset of `701.969px` was previously better than 702px.